autogpt_platform/frontend/src/components/agptui/CreatorCard.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
```

Comment:
imo seperate base level ui from using those items in stuff like specific ui's such as 'creator card'. What are y'all's thoughts